### ISSUE TITLE
Color contrast issue on Blog title

### ISSUE THREAD

**Issue by phucduong86** [phucduong86/phucduong86.github.io#3]
Page header "Phuc duong's blog" does not meet color contrast requirement on hover.

**Comment by phucduong86** [comment]
Issue resolved.
The link is now removed and the text is just heading text.
